device/thunder_ripple_sdk/src/processors/thunder_persistent_store.rs

```diff
@@ -92,6 +92,7 @@ impl ThunderStorageRequestProcessor {
             json!({
                 "namespace": data.namespace,
                 "key": data.key,
+                "scope": data.scope.unwrap_or_default(),
```

Comment:
so this will do an empty string for all the None cases?
Not sure we want that, shouldn't this just leave off scope if it is None?